src/utils.rs

```diff
@@ -1 +1,9 @@
-pub mod cast;
+use std::{slice, mem};
+
+/// Gives a mutable slice of the bytes of the given element.
+#[inline]
+pub fn bytes_of_mut<T: 'static + Copy>(elem: &mut T) -> &mut [u8] {
+    let slice = slice::from_mut(elem);
+    let new_len = mem::size_of_val(slice);
+    unsafe { slice::from_raw_parts_mut(slice.as_ptr() as *mut u8, new_len) }
```

Comment:
Not sure we need to have mutable `&[8]`, i think the bytes will never be modified

Reply:
It is required for the `ArpHeader::from_slice`method [to be able to copy the bytes from the source into this mutable slice of bytes](https://github.com/Lsh0x/EagleEyes/pull/18/files#diff-6d1443e173a8cddcab0d0e73b3106b98364ce747d8a290478e57edc0f3a080b6R22-R23) which corresponds to the structs bytes themselves.

Reply:
I see.